styles.css

```
@@ -1,6 +1,40 @@
 @charset 'UTF-8';
 
-h1 {
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 2.4rem;
+  font-size: 1.2rem;
   font-family: Arial, Helvetica, sans-serif;
-  color: blueviolet;
+  background-color: #f3f3f3;
+}
+
+nav a {
+  text-decoration: none;
+}
+
+a:link {
+  color: black;
+}
+
+a:visited {
+  color: black;
+}
```

```suggestion
nav a {
  text-decoration: none;
  color: black;
}
```
Nao sei se sao realmente necessarios todos esses estilos repetindo a mesma coisa. Acho que voce pode simplesmente adicionar `color: black'` no `nav a`

---

Realmente, @jhrodriguesit !
Valeu demais!